test/e2e/kubernetes/pod/pod.go

```diff
@@ -1026,3 +1045,20 @@ func (c *Container) getMemoryRequests() string {
 func (c *Container) getMemoryLimits() string {
 	return c.Resources.Limits.Memory
 }
+
+// getExternalURL returns an external URL that is not in the passed-in array of strings
+func getExternalURL(urls []string) string {
+	m := map[string]bool{}
+	for _, url := range externalURLs {
```

Comment:
the `externalURLs` var is only used here so it seems like it doesn't need to be a global var

Reply:
fair, the only reason I put it up there would be for easier maintenance, but we can make it private for now